spec/runners/crud/requirement.rb

```diff
@@ -106,6 +107,9 @@ def satisfied?
             ok = ok && [:allow, :forbid].include?(serverless)
           end
         end
+        if @auth
+          ok &&= (@auth == cc.auth_enabled?)
```

Comment:
This should simply be `ok &&= cc.auth_enabled?`, the extra equality condition may fail if auth_enabled? returns a truthy but not true value.

Reply:
👍